README.md

```diff
@@ -2,4 +2,90 @@
 
 [![Build Status](https://travis-ci.com/SpineEventEngine/bootstrap.svg?branch=master)](https://travis-ci.com/SpineEventEngine/bootstrap)
 
-Tools for bootstrapping projects built with Spine
+The Gradle plugin for bootstrapping projects built with Spine.
+
+## Applying to the project
+
+In order to apply to a single-module project, reference the plugin in the `plugins` block 
+in `build.gralde`:
+```gradle
+plugins {
+    id 'io.spine.tools.gradle.bootstrap' version '0.15.3'
+}
+```
+
+In order to apply to a multi-module project:
+ 1. declare the plugin in the root module:
+```gradle
+plugins {
+    id 'io.spine.tools.gradle.bootstrap' version '0.15.3' apply false
+}
+```
+ 2. apply the plugin in the `subprojects` section:
+```gradle
+subprojects {
+    apply plugin: 'io.spine.tools.gradle.bootstrap'
+}
+```
+Alternatively, apply the plugin only to some modules. In the `build.gradle` of the target module:
+```gradle
+apply plugin: 'io.spine.tools.gradle.bootstrap'
+```
+
```

Comment:
Please add reference to Gradle documentation on this “trick” of applying a plugin to a multi-module project.